src/app/components/BibPage/BibDetails_Functional.jsx

```diff
@@ -0,0 +1,118 @@
+import PropTypes from 'prop-types';
+import React, { useMemo } from 'react';
+import { isArray as _isArray, isEmpty as _isEmpty } from 'underscore';
+import { useBibParallel } from '../../context/Bib.Provider';
+import {
+  combineBibDetailsData,
+  groupNotesBySubject,
+  setParallelToNote,
+} from '../../utils/bibDetailsUtils';
+import LibraryItem from '../../utils/item';
+import DefinitionField from './components/DefinitionField';
+import DefinitionNoteField from './components/DefinitionNoteField';
+import DefinitionList from './DefinitionList';
+
+const BibDetails_Functional = ({ fields = [], marcs, resources }) => {
```

Comment:
looks like `<BibDetails>` is only being used in a `tabbedtest`.  Any reason to create a new `BibDetails_Functional` component rather than replace `BibDetails`?